app/models/concerns/contextualization.rb

```diff
@@ -25,7 +25,7 @@ def submission_fields_overrides
 
     delegate :visible_success_output?, to: :exercise
     delegate :output_content_type, to: :language
-    delegate :should_retry?, :to_submission_status, *Mumuki::Domain::Status::Submission.test_selectors, to: :submission_status
+    delegate :should_retry?, :to_submission_status, :completed?, *Mumuki::Domain::Status::Submission.test_selectors, to: :submission_status
```

Comment:
So I guess we're really using `completed?` :stuck_out_tongue:. I thought it was just a suggestion, an idea, based on your other comment.

Reply:
We are using `completed?` to make the assignment polymorphic with an indicator.
`completed?` answers if a certain progress item is, well, completed or not.

When I mentioned the thing about `passed?` I was talking about the places where we are asking it to show texts like "Congrats! You've done it". We want an skipped exercise to behave visually as a passed exercise, but it isn't technically in that status. So my idea was to have another method, like `approved?` (I realized that `completed?` is not a good one since we are using it with another purpose) and make it `passed? || skipped?`. It feels odd for an skipped exercise to answer true to `passed?`.